pylintrc_tweaks

```diff
@@ -2,3 +2,12 @@
 [MASTER]
 ignore = migrations
 load-plugins = edx_lint.pylint,pylint_django,pylint_celery
+
+[MESSAGES CONTROL]
+disable =
```

Comment:
`pylint==2.10.0` introduced these new warnings which are being disabled in edx repos.